docs/http-api/entry-api/delete_data.mdx

```diff
@@ -0,0 +1,324 @@
+---
+title: Delete Data
+description: HTTP Endpoints for deleting data from an entry in the database.
+---
+
+<head>
+  <link
+    rel="canonical"
+    href="https://www.reduct.store/docs/http-api/entry-api/delete_data"
+  />
+</head>
+
+# Entry API to Delete Data
+
+The API provides several endpoints for deleting data from an entry in the database. It can be an entire entry, a specific record, a batch of records in a single request or records within a time range.
+
+import SwaggerComponent from "@site/src/components/SwaggerComponent";
+
+<SwaggerComponent
+    method="DELETE"
+    path="/api/v1/b/:bucket_name/:entry_name"
+    summary="Remove a record from an entry or the entire entry"
+    description={
+        <>
+            The method removes a record from an entry or the entire entry.
+            If the `ts` query parameter is set, the record with the timestamp is removed.
+            Otherwise, the entire entry is removed.
```

Comment:
By reading the statement, I realized that it could be risky (if someone set ts to None accidentally). InfluxDb makes the time range mandatory: https://docs.influxdata.com/influxdb/v2/write-data/delete-data/
Maybe something for another release.

Reply:
just realized that you have the "Remove records within a time range" endpoint. Maybe that endpoint can also remove a single record (for example if start = stop), and the /api/v1/b/:bucket_name/:entry_name removes the entire entry

Reply:
Yes, it's dangerous if you use the HTTP API directly, but in the SDKs it's wrapped in methods. My intention was to keep the same structure of endpoints as for read and write operations, assuming that users would use SDKs or wrap the HTTP calls and test them.